### ISSUE TITLE
Implement Random

### ISSUE THREAD

**Issue by Jason2605** [dictu-lang/Dictu#273]
# Random
## Summary
Currently within Dictu there is no way to generate a random number. Along with the ability to generate random numbers it would be nice if we could both generate them between a range, and have the ability to select a random element of a list.

### Examples
```
import Random;

Random.random(); // Number between 0-1
Random.range(1, 10); // Number between start-end
Random.select([1, 2, 3]); // Randomly selected element from a list
```

**Comment by SeanMcLoughlin** [comment]
Hey, mind if I take this one?

**Comment by damn-dvlpr** [comment]
As per my convo with @Jason2605 , would want to work on this.

**Comment by Jason2605** [comment]
Closed as part of #274
